.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Package
+        run: cargo package
+
+      - name: Publish
+        run: |
+          cargo login ${{ secrets.TOKEN_CRATES_IO }}
```

Comment:
@arthurkhlghatyan  Please don't forget set your token for crates.io to this repository actions secrets.

Reply:
@arthurkhlghatyan Please help to set your token for crates.io? I have no permission to set it.

Reply:
Hey @fewensa token is set.

Reply:
Seems there's an issue with workflow configuration, if this is a quick fix feel free to open another PR otherwise I will check this in the morning.